browser/browser_context_keyed_service_factories.cc

```diff
@@ -104,6 +105,8 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
 #endif
 
   PermissionLifetimeManagerFactory::GetInstance();
+
+  skus::SkusServiceFactory::GetInstance();
```

Comment:
this should have a guard for the feature flag

Reply:
Guard added in https://github.com/brave/brave-core/pull/10847/commits/f1fd81200a1006b5c0971b6805968acf4934c79a